### PR TITLE
extra functionalities

### PR DESCRIPTION
**Pull request by harshitadaswani**
added remove all history feature
added delete a playlist feature
added create a playlist feature

### CODE REVIEW

**Comment by harshitadaswani** [comment]
deploy preview link: https://deploy-preview-7--crypto-library.netlify.app/

File: src/utils/ToastUtils.jsx [hunk 1]
```diff
@@ -20,3 +20,12 @@ export const removePlaylistToastFunction = () =>
```
Comment (sushil-mod) [inline src/utils/ToastUtils.jsx:20]: I like how implemented toast for showing updation messages

**Review by sushil-mod — APPROVED** [review]
Well-written code. Also, playlist functionality working fine

**Review by abhij1607 — APPROVED** [review]
Neat & clean code
Loved the UI
All feature works as intended

Can be merged

**Review by t007rushi — APPROVED** [review]
Everything some Looks fine
Naming is good
making constants is great one

**Pull request merged**